src/style.css

```diff
@@ -1 +1,72 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  height: 60px;
+  max-width: 1200px;
+  margin: 0 auto;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.moyo__logo {
+  margin: auto 50px;
+}
+
+.nav {
+  margin: auto;
+  margin-right: 50px;
+}
```

Comment:
Let's set 50px paddings in the header once, instead of setting 50px margins for logo and nav in 2 places.
Paddings fit better here